tests/unit/modules/junos_test.py

```diff
@@ -0,0 +1,1472 @@
+from __future__ import absolute_import
```

Comment:
Are we sure of keeping file name as junos_test? ideally it should be test_junos.